app/_hub/kong-inc/rate-limiting-advanced/index.md

```diff
@@ -7,10 +7,13 @@ version: 2.3.x
 
 desc: Upgrades Kong Rate Limiting with more flexibility and higher performance
 description: |
-  The Rate Limiting Advanced plugin for Kong Enterprise is a re-engineered
-  version of the incredibly popular Kong Community
-  [Rate Limiting plugin](/hub/kong-inc/rate-limiting/),
-  with greatly enhanced configuration options and performance.
+  The Rate Limiting Advanced plugin for Kong Enterprise is a re-engineered version of the Kong Community [Rate Limiting plugin](/hub/kong-inc/rate-limiting/).
+
+  As compared to the standard Rate Limiting plugin, Rate Limiting Advanced provides the additional configurations: `limit`, `window_size`, and `sync_rate`. It also has support for redis sentinel, redis cluster, and redis SSL.
+
+  In addition, the user has more rate limiting algorithms to choose from. These are more accurate as they enable configuration at a higher level of specificity. Learn more about our algorithms in [How to Design a Scalable Rate Limiting Algorithm](https://konghq.com/blog/how-to-design-a-scalable-rate-limiting-algorithm/).
+
+  Regarding performance, Rate Limiting Advanced has better throughput performance with better accuracy. Configure `sync_rate` to periodically sync with backend storage.
```

Comment:
It might be better to turn these three paragraphs into a list, something like:
* Additional configuration options and Redis Sentinel support: ...
* Algorithms:
* Performance: ...

Reply:
Done